frontend/src/metabase/entities/questions.js

```diff
@@ -16,11 +17,24 @@ import { canonicalCollectionId } from "metabase/collections/utils";
 import forms from "./questions/forms";
 import { updateIn } from "icepick";
 
+const listCards = GET("/api/card");
+const listReferencedCards = GET("/api/card/:questionId/referenced-cards");
```

Comment:
On the backend we call them cards, but on the frontend we call them questions. So maybe `listCards` should be 
`listQuestions` and `listReferencedCards` should be `listReferencedQuestions`?